doc/src/manual/conversion-and-promotion.md

```diff
@@ -175,6 +175,19 @@ argument: the argument name is omitted prior to the `::` symbol, and only the ty
 This is the syntax in Julia for a function argument whose type is specified but whose value
 does not need to be referenced by name.
 
+Note that implicit type conversion requires `import Base.convert`:
+
+```julia
+module Test
+  import Base.convert #necessary for implicit conversion
+  type MyType <: Number
+    n :: Int
+  end
+  #       DESIRED      GIVEN        HOW
```

Comment:
What does this comment mean?

Reply:
It's aligned with the following line as a key for beginners to understand the definition.

Reply:
This part of the manual is about `convert` though, not about how to write a function in the first place. I think that's already covered quite well in the chapters "Functions" and "Methods".